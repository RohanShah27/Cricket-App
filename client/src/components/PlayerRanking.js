import React, { Component } from "react";
import { getPlayerRanking, getTeamRanking } from "../actions/Rankings";
import { connect } from "react-redux";
import "../styles/PlayerRanking.css";
import playeruser from "./player.jpg";
export class PlayerRanking extends Component {
  constructor(props) {
    super(props);
    this.sendData = this.sendData.bind(this);
  }

  componentDidMount() {
    let ranking = {
      format: "Test"
    };
    this.props.getPlayerRanking(ranking);
    this.props.getTeamRanking(ranking);
  }

  state = {
    type: "",
    format: "",
    testClick: true,
    odiClick: false,
    t20Click: false
  };

  sendData(formats) {
    this.setState({
      format: formats
    });
    let ranking = {
      format: this.state.format
    };
    this.props.getPlayerRanking(ranking);
    this.props.getTeamRanking(ranking);
  }

  render() {
    let playerBatting = this.props.players.slice(0, 10);
    let playerBowling = this.props.players.slice(10, 20);
    let playerallRounder = this.props.players.slice(20, 30);
    console.log(this.props);
    return (
      <div>
        <div className="rankings-button-container">
          <button
            className={
              this.state.testClick
                ? "ranking-test-button ranking-active"
                : "ranking-test-button"
            }
            onClick={() => {
              this.sendData("Test");
              this.setState({
                testClick: true,
                odiClick: false,
                t20Click: false
              });
            }}
          >
            Test
          </button>
          <button
            className={
              this.state.odiClick ? " ranking-active" : "ranking-odi-button"
            }
            onClick={() => {
              this.sendData("ODI");
              this.setState({
                testClick: false,
                odiClick: true,
                t20Click: false
              });
            }}
          >
            ODI
          </button>
          <button
            className={
              this.state.t20Click ? "ranking-t20-active" : "ranking-t20-button"
            }
            onClick={() => {
              this.sendData("T20");
              this.setState({
                testClick: false,
                odiClick: false,
                t20Click: true
              });
            }}
          >
            T20
          </button>
        </div>
        <div className="rankingmain-container">
          <div class="ranking-container">
            {/* Table for Batting Stats for all Formats */}
            <div>
              <h2 className="rankings-container-header">
                {" "}
                ICC {this.state.format} Batting
              </h2>
              <table class="ranking-table">
                <tr>
                  <th>Rank</th>
                  <th>Player Image</th>
                  <th>Player</th>
                  <th>Team</th>
                  <th>Ratings</th>
                </tr>
                {playerBatting.map(player => (
                  <tr>
                    <td>{player.position}</td>
                    <td>
                      <img src={playeruser} className="rankingplayerIcon" />
                    </td>
                    <td>{player.player_name}</td>
                    <td>{player.player_team}</td>
                    <td>{player.ratings}</td>
                  </tr>
                ))}
              </table>
            </div>

            {/* Table for Bowling Stats for all Formats */}
            <div>
              <h2 className="rankings-container-header">
                ICC {this.state.format} Bowling
              </h2>
              <table class="ranking-table">
                <tr>
                  <th>Rank</th>
                  <th>Player Image</th>
                  <th>Player</th>
                  <th>Team</th>
                  <th>Ratings</th>
                </tr>
                {playerBowling.map(player => (
                  <tr>
                    <td>{player.position}</td>
                    <td>
                      <img src={playeruser} className="rankingplayerIcon" />
                    </td>
                    <td>{player.player_name}</td>
                    <td>{player.player_team}</td>
                    <td>{player.ratings}</td>
                  </tr>
                ))}
              </table>
            </div>
          </div>

          {/* Second row for ranking starts here */}

          <div class="ranking-container1">
            {/* Table for All-Rounder Stats for all Formats */}
            <div>
              <h2 className="rankings-container-header">
                ICC {this.state.format} All-Rounder
              </h2>
              <table class="ranking-table">
                <tr>
                  <th>Rank</th>
                  <th>Player Image</th>
                  <th>Player</th>
                  <th>Team</th>
                  <th>Ratings</th>
                </tr>
                {playerallRounder.map(player => (
                  <tr>
                    <td>{player.position}</td>
                    <td>
                      <img src={playeruser} className="rankingplayerIcon" />
                    </td>
                    <td>{player.player_name}</td>
                    <td>{player.player_team}</td>
                    <td>{player.ratings}</td>
                  </tr>
                ))}
              </table>
            </div>

            {/* Table for Team Stats for all Formats */}
            <div>
              <h2 className="rankings-container-header">
                ICC {this.state.format} Teams
              </h2>
              <table class="ranking-table">
                <tr>
                  <th>Rank</th>
                  <th>Team Image</th>
                  <th>Team</th>
                  <th>Ratings</th>
                  <th>Points</th>
                </tr>
                {this.props.teams.map(team => (
                  <tr>
                    <td>{team.position}</td>
                    <td>
                      <img src={playeruser} className="rankingplayerIcon" />
                    </td>
                    <td>{team.team_name}</td>
                    <td>{team.rating}</td>
                    <td>{team.points}</td>
                  </tr>
                ))}
              </table>
            </div>
          </div>
        </div>
      </div>
    );
  }
}

const mapStateToProps = state => ({
  players: state.playerRankingReducer.players,
  teams: state.teamRankingReducer.teams
});

export default connect(
  mapStateToProps,
  { getPlayerRanking, getTeamRanking }
)(PlayerRanking);
