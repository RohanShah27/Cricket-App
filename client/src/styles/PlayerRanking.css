@import url(https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css);
@import url(https://fonts.googleapis.com/css?family=Work+Sans);

.rankingplayerIcon {
  height: 30px;
}
.rankings-container-header {
  text-align: center;
}

.rankingmain-container {
  width: 90%;
  margin: 0px auto;
}

.ranking-container {
  display: flex;
  flex-direction: row;
  padding-top: 10px;
}

.ranking-table {
  border: 1px solid #c2c2c2;
  box-shadow: 0px 4px 6px 0px #c2c2c2;
  font-family: arial, sans-serif;
  border-collapse: collapse;
  width: 100%;
  margin-top: 20px;
}

.ranking-table th {
  border: 1px solid #dddddd;
  text-align: center;
  padding: 15px;
}

.ranking-table td {
  border: 1px solid #dddddd;
  text-align: center;
  padding: 15px;
}

.ranking-table tr:nth-child(even) {
  background-color: #dddddd;
}

.ranking-container > div,
.ranking-container1 > div {
  width: 100%;
  margin: 20px;
  text-align: left;
  font-size: 14px;
}
.ranking-container1 {
  display: flex;
  flex-direction: row;
  padding-top: 10px;
}

* {
  font-family: Work Sans, sans-serif;
  user-select: none;
}
.rankings-button-container {
  margin-left: 80px;
  padding-top: 20px;
  width: 350px;
  height: 70px;
}
.ranking-test-button {
  border-top-left-radius: 8px;
  border-bottom-left-radius: 8px;
  background-color: white;
  color: #f39c12;
  width: 100px;
  height: 50px;
  margin-left: 5px;
  border: 1px solid #f39c12;
}

.ranking-active {
  width: 100px;
  height: 50px;
  margin-left: 5px;
  border: 1px solid #fdf0dc;
  background-color: #f39c12;
  color: #ffffff;
  box-shadow: 0px 4px 6px 0px #c2c2c2;
  outline: none;
}

.ranking-t20-button {
  border-top-right-radius: 8px;
  border-bottom-right-radius: 8px;
  background-color: white;
  color: #f39c12;
  width: 100px;
  height: 50px;
  margin-left: 5px;
  border: 1px solid #f39c12;
}
.ranking-t20-active {
  width: 100px;
  height: 50px;
  margin-left: 5px;
  border: 1px solid #fdf0dc;
  background-color: #f39c12;
  border-top-right-radius: 8px;
  border-bottom-right-radius: 8px;
  color: #ffffff;
  box-shadow: 0px 4px 6px 0px #c2c2c2;
  outline: none;
}

.ranking-odi-button {
  border: 1px solid #f39c12;
  margin-left: 5px;
  background-color: white;
  color: #f39c12;
  width: 100px;
  height: 50px;
}

@media screen and (max-width: 768px) {
  .ranking-container {
    display: block;
  }
  .ranking-container1 {
    display: block;
  }
  .ranking-table {
    width: 70%;
  }
  .rankings-container-header {
    margin-right: 25vw;
  }
}
