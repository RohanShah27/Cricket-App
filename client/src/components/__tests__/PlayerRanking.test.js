import React from "react";
import { mount } from "enzyme";
import { PlayerRanking } from "../PlayerRanking";

const onClick = jest.fn();
// const sendData = jest.fn();
const playerRanking = jest.fn();
const getPlayerRanking = jest.fn();
const getTeamRanking = jest.fn();
let gender = "male";
const players = [];
for (var i = 0; i < 30; i++) {
  players.push({
    position: "test position",
    player_name: "test name",
    player_team: "test team",
    ratings: "test ratings"
  });
}
const teams = [
  {
    position: "test position",
    team_name: "test name",
    rating: "test rating",
    points: "test points"
  }
];
const wrapper = mount(
  <PlayerRanking
    playerRanking={playerRanking}
    players={players}
    teams={teams}
    gender={gender}
    getPlayerRanking={getPlayerRanking}
    getTeamRanking={getTeamRanking}
    onClick={onClick}
  />
);

describe("Test Team Component", () => {
  afterEach(() => {
    jest.clearAllMocks();
  });
  it("should render the component", () => {
    expect(wrapper).toMatchSnapshot();
  });
  it("should check for component did mount as a function", () => {
    const componentDidMount = jest.spyOn(
      PlayerRanking.prototype,
      "componentDidMount"
    );
    wrapper.instance().componentDidMount();
    expect(componentDidMount).toHaveBeenCalled();
    componentDidMount.mockClear();
  });

  it("should check mapping of playerbatting Position array", () => {
    expect(wrapper.find("#battingPosition0").props().children).toBe(
      "test position"
    );
  });
  it("should check mapping of playerbatting Position array", () => {
    expect(wrapper.find("#battingName0").props().children).toBe("test name");
  });
  it("should check mapping of playerbatting Position array", () => {
    expect(wrapper.find("#battingTeam0").props().children).toBe("test team");
  });
  it("should check mapping of playerbatting Position array", () => {
    expect(wrapper.find("#battingRating0").props().children).toBe(
      "test ratings"
    );
  });
  it("should check mapping of playerbowling Position array", () => {
    expect(wrapper.find("#bowlingPosition0").props().children).toBe(
      "test position"
    );
  });
  it("should check mapping of playerbowling Position array", () => {
    expect(wrapper.find("#bowlingName0").props().children).toBe("test name");
  });
  it("should check mapping of playerbowling Position array", () => {
    expect(wrapper.find("#bowlingTeam0").props().children).toBe("test team");
  });
  it("should check mapping of playerbowling Position array", () => {
    expect(wrapper.find("#bowlingRating0").props().children).toBe(
      "test ratings"
    );
  });
  it("should check mapping of All Rounder Position array", () => {
    expect(wrapper.find("#allRounderPosition0").props().children).toBe(
      "test position"
    );
  });
  it("should check mapping of All Rounder Position array", () => {
    expect(wrapper.find("#allRounderName0").props().children).toBe("test name");
  });
  it("should check mapping of All Rounder Position array", () => {
    expect(wrapper.find("#allRounderTeam0").props().children).toBe("test team");
  });
  it("should check mapping of All Rounder Position array", () => {
    expect(wrapper.find("#allRounderRating0").props().children).toBe(
      "test ratings"
    );
  });
  it("should simulate button click on Test button tab ", () => {
    // const sendData = jest.spyOn(PlayerRanking.prototype, "sendData");
    // console.log(sendData);
    wrapper.find("#testTab").simulate("click");
    // expect(sendData).toHaveBeenCalledWith("Test");
    expect(wrapper.state().testClick).toBe(true);
    expect(wrapper.state().t20Click).toBe(false);
    expect(wrapper.state().odiClick).toBe(false);
  });
  it("should simulate button click on ODI button tab ", () => {
    // const sendData = jest.spyOn(PlayerRanking.prototype, "sendData");

    wrapper.find("#odiTab").simulate("click");
    expect(wrapper.state().testClick).toBe(false);
    expect(wrapper.state().t20Click).toBe(false);
    expect(wrapper.state().odiClick).toBe(true);
    // expect(sendData).toBeCalledWith("ODI");
  });
  it("should simulate button click on T20 button tab ", () => {
    wrapper.find("#t20Tab").simulate("click");
    expect(wrapper.state().testClick).toBe(false);
    expect(wrapper.state().t20Click).toBe(true);
    expect(wrapper.state().odiClick).toBe(false);
    // expect(sendData).toBeCalledWith("T20");
  });
  it("should check mapping of Team Position array", () => {
    expect(wrapper.find("#teamPosition0").props().children).toBe(
      "test position"
    );
  });
  it("should check mapping of Team Name array", () => {
    expect(wrapper.find("#teamName0").props().children).toBe("test name");
  });
  it("should check mapping of Team Rating array", () => {
    expect(wrapper.find("#teamRating0").props().children).toBe("test rating");
  });
  it("should check mapping of Team Points array", () => {
    expect(wrapper.find("#teamPoints0").props().children).toBe("test points");
  });

  it("should simulate test button", () => {
    expect(
      wrapper
        .find("button")
        .at(0)
        .simulate("click")
    );
  });
  it("should simulate odi button", () => {
    expect(
      wrapper
        .find("button")
        .at(1)
        .simulate("click")
    );
  });
});
