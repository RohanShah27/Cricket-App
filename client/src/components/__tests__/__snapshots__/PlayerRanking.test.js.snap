// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Team Component should render the component 1`] = `
<PlayerRanking
  gender="male"
  getPlayerRanking={
    [MockFunction] {
      "calls": Array [
        Array [
          Object {
            "format": "Test",
            "gender": "male",
          },
        ],
      ],
      "results": Array [
        Object {
          "type": "return",
          "value": undefined,
        },
      ],
    }
  }
  getTeamRanking={
    [MockFunction] {
      "calls": Array [
        Array [
          Object {
            "format": "Test",
            "gender": "male",
          },
        ],
      ],
      "results": Array [
        Object {
          "type": "return",
          "value": undefined,
        },
      ],
    }
  }
  onClick={[MockFunction]}
  playerRanking={[MockFunction]}
  players={
    Array [
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
      Object {
        "player_name": "test name",
        "player_team": "test team",
        "position": "test position",
        "ratings": "test ratings",
      },
    ]
  }
  teams={
    Array [
      Object {
        "points": "test points",
        "position": "test position",
        "rating": "test rating",
        "team_name": "test name",
      },
    ]
  }
>
  <div>
    <div
      className="rankings-button-container"
    >
      <button
        className="ranking-test-button ranking-active"
        id="testTab"
        onClick={[Function]}
      >
        Test
      </button>
      <button
        className="ranking-odi-button"
        id="odiTab"
        onClick={[Function]}
      >
        ODI
      </button>
      <button
        className="ranking-t20-button"
        id="t20Tab"
        onClick={[Function]}
      >
        T20
      </button>
    </div>
    <div
      className="rankingmain-container"
    >
      <div
        class="ranking-container"
      >
        <div>
          <h2
            className="rankings-container-header"
          >
             
            ICC 
             Batting
          </h2>
          <table
            class="ranking-table"
          >
            <tr>
              <th>
                Rank
              </th>
              <th>
                Player
              </th>
              <th>
                Team
              </th>
              <th>
                Ratings
              </th>
            </tr>
            <tr>
              <td
                id="battingPosition0"
              >
                test position
              </td>
              <td
                id="battingName0"
              >
                test name
              </td>
              <td
                id="battingTeam0"
              >
                test team
              </td>
              <td
                id="battingRating0"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="battingPosition1"
              >
                test position
              </td>
              <td
                id="battingName1"
              >
                test name
              </td>
              <td
                id="battingTeam1"
              >
                test team
              </td>
              <td
                id="battingRating1"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="battingPosition2"
              >
                test position
              </td>
              <td
                id="battingName2"
              >
                test name
              </td>
              <td
                id="battingTeam2"
              >
                test team
              </td>
              <td
                id="battingRating2"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="battingPosition3"
              >
                test position
              </td>
              <td
                id="battingName3"
              >
                test name
              </td>
              <td
                id="battingTeam3"
              >
                test team
              </td>
              <td
                id="battingRating3"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="battingPosition4"
              >
                test position
              </td>
              <td
                id="battingName4"
              >
                test name
              </td>
              <td
                id="battingTeam4"
              >
                test team
              </td>
              <td
                id="battingRating4"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="battingPosition5"
              >
                test position
              </td>
              <td
                id="battingName5"
              >
                test name
              </td>
              <td
                id="battingTeam5"
              >
                test team
              </td>
              <td
                id="battingRating5"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="battingPosition6"
              >
                test position
              </td>
              <td
                id="battingName6"
              >
                test name
              </td>
              <td
                id="battingTeam6"
              >
                test team
              </td>
              <td
                id="battingRating6"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="battingPosition7"
              >
                test position
              </td>
              <td
                id="battingName7"
              >
                test name
              </td>
              <td
                id="battingTeam7"
              >
                test team
              </td>
              <td
                id="battingRating7"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="battingPosition8"
              >
                test position
              </td>
              <td
                id="battingName8"
              >
                test name
              </td>
              <td
                id="battingTeam8"
              >
                test team
              </td>
              <td
                id="battingRating8"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="battingPosition9"
              >
                test position
              </td>
              <td
                id="battingName9"
              >
                test name
              </td>
              <td
                id="battingTeam9"
              >
                test team
              </td>
              <td
                id="battingRating9"
              >
                test ratings
              </td>
            </tr>
          </table>
        </div>
        <div>
          <h2
            className="rankings-container-header"
          >
            ICC 
             Bowling
          </h2>
          <table
            class="ranking-table"
          >
            <tr>
              <th>
                Rank
              </th>
              <th>
                Player
              </th>
              <th>
                Team
              </th>
              <th>
                Ratings
              </th>
            </tr>
            <tr>
              <td
                id="bowlingPosition0"
              >
                test position
              </td>
              <td
                id="bowlingName0"
              >
                test name
              </td>
              <td
                id="bowlingTeam0"
              >
                test team
              </td>
              <td
                id="bowlingRating0"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="bowlingPosition1"
              >
                test position
              </td>
              <td
                id="bowlingName1"
              >
                test name
              </td>
              <td
                id="bowlingTeam1"
              >
                test team
              </td>
              <td
                id="bowlingRating1"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="bowlingPosition2"
              >
                test position
              </td>
              <td
                id="bowlingName2"
              >
                test name
              </td>
              <td
                id="bowlingTeam2"
              >
                test team
              </td>
              <td
                id="bowlingRating2"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="bowlingPosition3"
              >
                test position
              </td>
              <td
                id="bowlingName3"
              >
                test name
              </td>
              <td
                id="bowlingTeam3"
              >
                test team
              </td>
              <td
                id="bowlingRating3"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="bowlingPosition4"
              >
                test position
              </td>
              <td
                id="bowlingName4"
              >
                test name
              </td>
              <td
                id="bowlingTeam4"
              >
                test team
              </td>
              <td
                id="bowlingRating4"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="bowlingPosition5"
              >
                test position
              </td>
              <td
                id="bowlingName5"
              >
                test name
              </td>
              <td
                id="bowlingTeam5"
              >
                test team
              </td>
              <td
                id="bowlingRating5"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="bowlingPosition6"
              >
                test position
              </td>
              <td
                id="bowlingName6"
              >
                test name
              </td>
              <td
                id="bowlingTeam6"
              >
                test team
              </td>
              <td
                id="bowlingRating6"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="bowlingPosition7"
              >
                test position
              </td>
              <td
                id="bowlingName7"
              >
                test name
              </td>
              <td
                id="bowlingTeam7"
              >
                test team
              </td>
              <td
                id="bowlingRating7"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="bowlingPosition8"
              >
                test position
              </td>
              <td
                id="bowlingName8"
              >
                test name
              </td>
              <td
                id="bowlingTeam8"
              >
                test team
              </td>
              <td
                id="bowlingRating8"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="bowlingPosition9"
              >
                test position
              </td>
              <td
                id="bowlingName9"
              >
                test name
              </td>
              <td
                id="bowlingTeam9"
              >
                test team
              </td>
              <td
                id="bowlingRating9"
              >
                test ratings
              </td>
            </tr>
          </table>
        </div>
      </div>
      <div
        class="ranking-container1"
      >
        <div>
          <h2
            className="rankings-container-header"
          >
            ICC 
             All-Rounder
          </h2>
          <table
            class="ranking-table"
          >
            <tr>
              <th>
                Rank
              </th>
              <th>
                Player
              </th>
              <th>
                Team
              </th>
              <th>
                Ratings
              </th>
            </tr>
            <tr>
              <td
                id="allRounderPosition0"
              >
                test position
              </td>
              <td
                id="allRounderName0"
              >
                test name
              </td>
              <td
                id="allRounderTeam0"
              >
                test team
              </td>
              <td
                id="allRounderRating0"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="allRounderPosition1"
              >
                test position
              </td>
              <td
                id="allRounderName1"
              >
                test name
              </td>
              <td
                id="allRounderTeam1"
              >
                test team
              </td>
              <td
                id="allRounderRating1"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="allRounderPosition2"
              >
                test position
              </td>
              <td
                id="allRounderName2"
              >
                test name
              </td>
              <td
                id="allRounderTeam2"
              >
                test team
              </td>
              <td
                id="allRounderRating2"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="allRounderPosition3"
              >
                test position
              </td>
              <td
                id="allRounderName3"
              >
                test name
              </td>
              <td
                id="allRounderTeam3"
              >
                test team
              </td>
              <td
                id="allRounderRating3"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="allRounderPosition4"
              >
                test position
              </td>
              <td
                id="allRounderName4"
              >
                test name
              </td>
              <td
                id="allRounderTeam4"
              >
                test team
              </td>
              <td
                id="allRounderRating4"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="allRounderPosition5"
              >
                test position
              </td>
              <td
                id="allRounderName5"
              >
                test name
              </td>
              <td
                id="allRounderTeam5"
              >
                test team
              </td>
              <td
                id="allRounderRating5"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="allRounderPosition6"
              >
                test position
              </td>
              <td
                id="allRounderName6"
              >
                test name
              </td>
              <td
                id="allRounderTeam6"
              >
                test team
              </td>
              <td
                id="allRounderRating6"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="allRounderPosition7"
              >
                test position
              </td>
              <td
                id="allRounderName7"
              >
                test name
              </td>
              <td
                id="allRounderTeam7"
              >
                test team
              </td>
              <td
                id="allRounderRating7"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="allRounderPosition8"
              >
                test position
              </td>
              <td
                id="allRounderName8"
              >
                test name
              </td>
              <td
                id="allRounderTeam8"
              >
                test team
              </td>
              <td
                id="allRounderRating8"
              >
                test ratings
              </td>
            </tr>
            <tr>
              <td
                id="allRounderPosition9"
              >
                test position
              </td>
              <td
                id="allRounderName9"
              >
                test name
              </td>
              <td
                id="allRounderTeam9"
              >
                test team
              </td>
              <td
                id="allRounderRating9"
              >
                test ratings
              </td>
            </tr>
          </table>
        </div>
        <div>
          <h2
            className="rankings-container-header"
          >
            ICC 
             Teams
          </h2>
          <table
            class="ranking-table"
          >
            <tr>
              <th>
                Rank
              </th>
              <th>
                Team
              </th>
              <th>
                Ratings
              </th>
              <th>
                Points
              </th>
            </tr>
            <tr>
              <td
                id="teamPosition0"
              >
                test position
              </td>
              <td
                id="teamName0"
              >
                test name
              </td>
              <td
                id="teamRating0"
              >
                test rating
              </td>
              <td
                id="teamPoints0"
              >
                test points
              </td>
            </tr>
          </table>
        </div>
      </div>
    </div>
  </div>
</PlayerRanking>
`;
